src/ncp/ncp_base.cpp

```diff
@@ -2710,6 +2710,14 @@ template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_PHY_CALIBRATED_POWER>
 }
 #endif // OPENTHREAD_CONFIG_PLATFORM_POWER_CALIBRATION_ENABLE
 
+#if OPENTHREAD_FTD || OPENTHREAD_MTD
+template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_RADIO_CAPS>(void)
```

Comment:
It seems this is specifically for radio caps, should we rename it to `SPINEL_PROP_COPROCESSOR_CAPS`? Or we create a new property with that name?

Reply:
I think we can probably use the existing `SPINEL_PROP_CAPS`. And there are already some values that can help check if it's NCP:
```
    SPINEL_CAP_CONFIG__BEGIN = 32,                                                                                                                                                                                                                           
    SPINEL_CAP_CONFIG_FTD    = (SPINEL_CAP_CONFIG__BEGIN + 0),                                                                                                                                                                                               
    SPINEL_CAP_CONFIG_MTD    = (SPINEL_CAP_CONFIG__BEGIN + 1),                                                                                                                                                                                               
    SPINEL_CAP_CONFIG_RADIO  = (SPINEL_CAP_CONFIG__BEGIN + 2),                                                                                                                                                                                               
    SPINEL_CAP_CONFIG__END   = 40,  
```
I'll further check if we can just use this prop and keep radio capability unchanged.

Reply:
I think we can probably use the existing `SPINEL_PROP_CAPS`. And there are already some values that can help check if it's NCP:
```
    SPINEL_CAP_CONFIG__BEGIN = 32,                                                                                                                                                                                                                           
    SPINEL_CAP_CONFIG_FTD    = (SPINEL_CAP_CONFIG__BEGIN + 0),                                                                                                                                                                                               
    SPINEL_CAP_CONFIG_MTD    = (SPINEL_CAP_CONFIG__BEGIN + 1),                                                                                                                                                                                               
    SPINEL_CAP_CONFIG_RADIO  = (SPINEL_CAP_CONFIG__BEGIN + 2),                                                                                                                                                                                               
    SPINEL_CAP_CONFIG__END   = 40,  
```
I'll further check if we can just use this prop and keep radio capability unchanged.

Reply:
Yes. `SPINEL_PROP_CAPS` make sense for this.No need to add new properties.